webapp/src/pages/web/auth/login.tsx

```diff
@@ -1,24 +1,29 @@
 import Head from 'next/head';
 import { LoginForm } from 'dogma/features/auth/LoginForm';
-import { useAppSelector } from 'dogma/store';
+import { useAppDispatch, useAppSelector } from 'dogma/store';
 import { useEffect } from 'react';
 import { useRouter } from 'next/router';
+import { login } from 'dogma/features/auth/authSlice';
 
 const LoginPage = () => {
```

Comment:
Suggestion: What do you think of adding a convention that confines filenames under `pages/` to ends `Page`? This convention make the filename more readable and easy to navigate with file browsers. 
For example: `login.tsx` -> `LoginPage.tsx`, `projects.tsx` -> `ProjectsPage.tsx`